docs/internals/contributing/writing-code/coding-style.txt

```diff
@@ -215,8 +215,37 @@ Imports
 Template style
 ==============
 
-* In Django template code, put one (and only one) space between the curly
-  brackets and the tag contents.
+* In Django template code: 
+
+  The ``{% extends %}`` should be the first non-comment line
+
+  Do this:
+
+  .. code-block:: html+django
+
+      {% extends %}
+
+      Other code goes here
```

Comment:
Let’s replace this with a realistic second line:

```suggestion
      {% block content %}
      ...
```

...and in below examples.